packages/@okta/vuepress-site/guides/sign-into-your-web-app/before-you-begin/index.md

```diff
@@ -11,18 +12,12 @@ This guide walks you through how to sign users into your existing web app. It is
 ## Before You Begin
 These instructions assume that you: 
 
-* Are building Web app that's rendered by a server, otherwise see [Sign Users into a Single-Page App](sign-into-spa). If you are building a mobile app, see [Sign Users into a Mobile App](sign-into-mobile-app). If you are building an API, see [Protect an API].
+* Are building a web app that's rendered by a server, otherwise see [Sign Users into a Single-Page App](sign-into-spa). If you are building a mobile app, see [Sign Users into a Mobile App](sign-into-mobile-app). If you are building an API, see [Protect an API].
```

Comment:
Can we make this into a two-level list?


* Are building a web app that's rendered by a server. 
  * If you are building a web app that's rendered by a client, see [Sign Users into a Single-Page App](sign-into-spa). 
  * If you are building a mobile app, see [Sign Users into a Mobile App](sign-into-mobile-app). 
  * If you are building an API, see [Protect an API].

Reply:
These are actually commented out right now. So, we're going to add them back in later when all of the guides are done.